css/client.css

```diff
@@ -638,3 +638,111 @@ input:checked + .slider:before {
 .tabHeader {
     background: var(--medDark);
 }
+
+/* style for wordlinks */
```

Comment:
Could you add a distinct class for word links so that it's less likely to collide with other CSS? e.g. add "wordlinks" class to the top element and prepend all these CSS sections with .wordlinks.